lib/tasks/developer_feature_update.rake

```diff
@@ -0,0 +1,9 @@
+namespace :developers do
+  desc "Find and notify developer on new feature updates"
+  task notify_product_feature_update: :environment do
+    developers = Developer.actively_looking_or_open.visible.product_feature_notifications
+    developers.each(&:notify_product_feature_update)
```

Comment:
Let's use `#find_each` so Rails will find these in batches and decrease the load on the database.